tool/teleport/common/teleport.go

```diff
@@ -422,10 +409,12 @@ func Run(options Options) (app *kingpin.Application, executedCommand string, con
 	joinOpenSSH.Flag("join-method", "Method to use to join the cluster (token, iam, ec2)").EnumVar(&ccf.JoinMethod, "token", "iam", "ec2")
 	joinOpenSSH.Flag("openssh-config", "Path to the OpenSSH config file").Default("/etc/ssh/sshd_config").StringVar(&ccf.OpenSSHConfigPath)
 	joinOpenSSH.Flag("openssh-keys-path", "Path to the place teleport keys and certs").Default(agentlessKeysDir).StringVar(&ccf.OpenSSHKeysPath)
+	joinOpenSSH.Flag("openssh-keys-backup-path", "Path to the place teleport keys and certs should be backed up to after certificate rotation").Default(agentlessKeysBackupDir).StringVar(&ccf.OpenSSHKeysBackupPath)
 	joinOpenSSH.Flag("restart-sshd", "Restart OpenSSH").Default("true").BoolVar(&ccf.RestartOpenSSH)
 	joinOpenSSH.Flag("insecure", "Insecure mode disables certificate validation").BoolVar(&ccf.InsecureMode)
 	joinOpenSSH.Flag("additional-principals", "Comma separated list of host names the node can be accessed by").StringVar(&ccf.AdditionalPrincipals)
 	joinOpenSSH.Flag("debug", "Enable verbose logging to stderr.").Short('d').BoolVar(&ccf.Debug)
+	joinOpenSSH.Flag("rotate", "Optionally Rotate the CA certs (update_servers, rollback)").EnumVar(&ccf.RotateOpenSSH, types.RotationPhaseUpdateServers, types.RotationPhaseRollback)
```

Comment:
This flag help message is confusing. I would rephrase to `Optionally specify the CA rotation phase to execute. Can be one of: update_servers (xxx) or rollback (xxx)` and fill in "xxx" with brief description of what will happen during this phase.